libcloudforensics/providers/aws/internal/ebs.py

```diff
@@ -204,17 +231,33 @@ def Copy(self, kms_key_id=None):
       copy_args['KmsKeyId'] = kms_key_id
     try:
       response = client.copy_snapshot(**copy_args)
-      return AWSSnapshot(
-          # If the call was successful, the response contains the new
-          # snapshot ID
-          response['SnapshotId'],
-          self.volume,
-          name='{0:s}-copy'.format(self.snapshot_id)
-      )
     except client.exceptions.ClientError as exception:
       raise RuntimeError('Could not copy snapshot {0:s}: {1:s}'.format(
           self.snapshot_id, str(exception)))
 
+    snapshot_copy = AWSSnapshot(
+        # If the call above was successful, the response contains the new
+        # snapshot ID
+        response['SnapshotId'],
```

Comment:
You're saying "if the call above was successful", but what happens if it isn't? If the answer to that is that it would be caught in the exception on L234, then it's enough to say "The response contains the new snapshot ID"

Reply:
Ah, this was in the `try` block before hence the comment. I removed it. L234 will indeed catch an exception is something goes bad.